### DISCUSSION TITLE
I have no idea if I've submitted the information for my SP-API-created FBA shipment properly. Help!

### DISCUSSION THREAD

**Discussion by dartheditous** [amzn/selling-partner-api-models#3504] [General]
Hi all,

I'm trying to complete my first pure SP-API FBA shipment and I'm struggling to understand if I've done it properly. I've asked Seller Support to check but have had no reply, and Seller Forums are also inaccessible to me for some reason (page reloads endlessly).

Creating and updating (because of removed items) the shipment was fine, but I'm confused about submitting the tracking and carton information (this is for a non-partnered LTL boxed shipment).

I uploaded a POST_FBA_INBOUND_CARTON_CONTENTS feed and after a few minutes the processing status changed to DONE. When I went to Seller Central to check (using the Old Workflow), it showed a table asking for box weights and dimensions, which was new to me (previous I would put this information in the packing contents spreadsheet) and the number of rows was correct. However the "Complete Shipment" button is greyed out until I fill in the table (which obviously I won't be able to do using SP-API).

But when I use getLabels, none of the labels show sku or quantity information. When I was doing everything via Seller Central, uploading an XLS packing list, once I had done this, the labels I got would show an Sku and a quantity if there was just one item in each box. But via SP-API, this doesn't seem to be the case. So I don't get this extra confirmation that everything's correct.

The only place where I can see that I may have strayed from the suggested SP-API workflow is that I uploaded the Carton Contents feed BEFORE uploading tracking numbers (using putTransportDetails - I know this worked, at least, because I can see the info when I use getTransportDetails).

It's also concerning me that I can apparently request any number of labels - up to 1000 - for my shipment and SP-API doesn't complain.

So my questions are:

a) Is seeing processingStatus = DONE the only way to confirm that my Contents feed has been successfully uploaded and applied to my shipment? Does this DEFINITELY confirm that my shipment contents have been accepted?
a) Is it normal that Sku information (when there is just one item in a box) does not appear on SP-API-generated labels, as it does with Seller-Central-downloaded labels?
b) Why is there a weight/dimensions table in Seller Central, which stop me from pressing Complete Shipment until I fill it in? There doesn't seem to be an equivalent to this via SP-API.

Thanks for your help!

**Comment by dartheditous** [comment]
Apologies, I'm on a deadline and was desperate scrabbling for answers. Anyway I've now learned how to get a feed processing report and this showed me where I was going wrong.

It's confusing that, despite the feed erroring, Amazon was still updating my number of boxes.

However if anyone can confirm that there is no way and no need to upload box weight and dimensions via SP-API (despite the table appearing in Seller Central), it would be much appreciated!

**Reply by homebuildersh** [reply]
did you end up getting this to work correctly? Meaning, I'm assuming that you are creating via SP/API an inbound shipment. Have you successfully been able to do so? If you did, can you please explain how got around the dimensions/weight question?

In addition, did you get as far as printing the shipping/FBA labels?

THANK YOU!

Best Regards,
